### PR TITLE
 feat(#8511): add gh action to run cht-conf 

### PR DESCRIPTION
**Pull request by garethbowen**
# Description

[description]

medic/cht-core#8511

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs

If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8511-cht-conf-action/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8511-cht-conf-action/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8511-cht-conf-action/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by garethbowen** [comment]
@kennsippell Can I get a review for this?

The main change is switching from the deprecated `npx medic-conf` to `npx cht` but it also updates the naming.

One alternative is to keep the action name the same which would automatically update everyone who uses it, but that would be a breaking change for users on old versions of medic-conf, so this is safer, albeit annoying for everyone to upgrade to.

File: .github/actions/deploy-conf/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+## Example GitHub Step
+
+```
+name: Example GitHub Workflow yml
+
+on: ['deployment']
+
+jobs:
+  deployment:
+    runs-on: ubuntu-22.04
```
Comment (kennsippell) [inline .github/actions/deploy-conf/README.md:16]: i think we use `runs-on: ubuntu-latest` for most deployment scripts now.  we might consider that  here also so this doc doesn't need upkeep

**Review by kennsippell — APPROVED** [review]
Looking reasonable.  I've opened https://github.com/medic/app-services-team/issues/574 to track the move for existing repositories.

File: .github/actions/deploy-conf/README.md (hunk 1 above)
Comment (garethbowen) [reply .github/actions/deploy-conf/README.md:16]: Thanks, done!

**Pull request merged**